boxtree/tools.py

```diff
@@ -37,11 +36,12 @@
 from loopy.version import LOOPY_USE_LANGUAGE_VERSION_2018_2  # noqa
 
 from functools import partial
+import sys
 
 
 # Use offsets in VectorArg by default.
 VectorArg = partial(_VectorArg, with_offset=True)
-
+ScalarArg = ScalarArg
```

Comment:
:thinking:

Reply:
LOL. I don't what this is doing. Removed.